addon/components/combo-box.js

```diff
@@ -980,6 +963,26 @@ export default Component.extend({
       'mobileDropdownVisible': true,
       '_oldInputValue': this.get('inputValue')
     });
+    if (isPresent(this.get('lazyCallback')) && this.get('lazyCallbackInProgress') === false) {
+      this._resetLazyCombobox();
+    }
+
+    schedule('afterRender', this, function() {
+      let that = this;
+      $(this.element).find('.dropdown').on('touchmove.mobilePagination', function () {
+        debounce(that, () => {
```

Comment:
This won't work as intended - `debounce` needs to have the same function (the same reference) as an input parameter to work properly. It the moment, you are creating new function every time  touchmove event is handled. Extract the function you want to debounce into a separate variable and pass that variable into debounce.